assets/components/src/notice/index.js

```diff
@@ -52,7 +52,9 @@ class Notice extends Component {
 		return (
 			<div className={ classes }>
 				{ noticeIcon }
-				<RawHTML className="newspack-notice__content">{ noticeText }</RawHTML>
+				<div className="newspack-notice__content">
+					{ !! rawHTML ? <RawHTML>{ noticeText }</RawHTML> : noticeText }
```

Comment:
nitpick: in a ternary expression, the condition will be treated as a boolean anyway
```suggestion
					{ rawHTML ? <RawHTML>{ noticeText }</RawHTML> : noticeText }
```

Also, I wonder if the rendering condition is necessary - a string in `RawHTML` will not cause any issues 🤔

Reply:
Fixed in 194104e